pkg/apis/serving/v1alpha1/local_model_cache_status.go

```diff
@@ -43,9 +43,6 @@ const (
 	NodeDownloading     NodeStatus = "NodeDownloading"
 	NodeDownloaded      NodeStatus = "NodeDownloaded"
 	NodeDownloadError   NodeStatus = "NodeDownloadError"
-	NodeDeleting        NodeStatus = "NodeDeleting"
```

Comment:
Remove these as they are not used, @greenmoon55 to confirm

Reply:
yeah, they are not used.